mlflow/server/js/src/common/utils/Utils.tsx

```diff
@@ -305,18 +308,26 @@ class Utils {
     const bitbucketMatch = sourceName.match(Utils.getBitbucketRegex());
     const gitMatch = sourceName.match(Utils.getGitRegex());
     let url = null;
-    if (gitHubMatch || gitLabMatch) {
-      const baseUrl = gitHubMatch ? 'https://github.com/' : 'https://gitlab.com/';
-      const match = gitHubMatch || gitLabMatch;
+    if (gitHubMatch) {
       url =
-        baseUrl +
-        match[1] +
+        'https://github.com/' +
+        gitHubMatch[1] +
         '/' +
-        match[2].replace(/.git/, '') +
+        gitHubMatch[2].replace(/.git/, '') +
         '/tree/' +
         sourceVersion +
         '/' +
-        match[3];
+        gitHubMatch[3];
+    } else if (gitLabMatch) {
+      url =
```

Comment:
Can we use an f-string instead of the concatenation?

Reply:
yep made the change - this concatenation was used originally, but f-string would be cleaner